src/overview/design-goals.md

```diff
@@ -43,3 +44,26 @@ We want to follow some conservative design heuristics to minimize risk and mista
 - Rely as much as possible on design components that are already proven in production environments.
 - Rely as much as possible on design components with adequate theoretical underpinnings and security analyses.
 - Minimize changes or variations on the above: strive to only alter existing work when necessary for overall design goals. For example, Zcash's privacy or issuance constraints are likely less common among existing PoS designs.
+
+
+# Non-goals
+
+These are not goals of the TFL design, either to simplify the scope of the initial design (aka [Out of Scope Goals](#out-of-scope-goals)), or because we believe some potential goal _should not_ be supported (aka [Anti-goals](#anti-goals)).
+
+## Out of Scope Goals
+
+While these desiderata may be common across the blockchain consensus design space, they are not specific goals for the initial TFL design. Note that these may be goals for future protocol improvements.
+
+- Prioritizing minimal time-to-finality over other considerations (such as protocol simplicity, impact on existing use cases, or other goals above).
+- In-protocol liquid staking derivatives.
+- Maximizing the PoS staked-voter count ceiling. For example, [Tendermint BFT](https://github.com/tendermint/tendermint#research) has a relatively low ceiling of ~hundreds of staked voters, whereas Ethereum's [Gasper](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/gasper/) supports hundreds of thousands of staked voters.
+
```

Comment:
- Reducing energy usage. While this would presumably be a goal of a pure PoS transition, it likely cannot be achieved for hybrid PoW/PoS without loss of security.